assets/src/styles/blocks/CarouselHeader/SidebarSlidesEditor.scss

```diff
@@ -0,0 +1,138 @@
+// Doing this for a more specificity
+.sidebar-slides {
+  position: relative;
+}
+
+.sidebar-slide {
+  pointer-events: none;
+
+  .components-panel__arrow {
+    display: none;
+  }
+
+  .components-panel__body-toggle {
+    pointer-events: none;
+    padding-right: 16px;
+    padding-left: 9px;
+  }
+
+  &.cloned-slide {
```

Comment:
[This part](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/729/files#diff-b11e1bdfff8bc0685a960c674ccbed0d863e5c63327f1c2a0d93f65333b98290R19-R46) doesn't need to be "inside of" `.sidebar-slide` to work. Would save a few bytes in the compiled CSS.